src/script/assets/AssetMapper.js

```diff
@@ -17,18 +17,15 @@
  *
  */
 
-window.z = window.z || {};
-window.z.assets = z.assets || {};
-
-z.assets.AssetMapper = {
+class AssetMapper {
```

Comment:
I believe this also doesn't need to be a class but rather individually exported functions.